interface/fax/fax_dispatch.php

```diff
@@ -463,7 +463,7 @@ function mergeTiffs()
 <html>
```

Comment:
Please add a todo for this file to modernize it as it is using table elements that aren't responsive here.